### ISSUE TITLE
Adding location of hands to "both hands" play area 

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#157]
We would like to add a sentence to the "both hands" indicating where the hands are located. Its a single sentence that follows the same logic as the dynamic scene summary bullet points for all three views of the sim. Here are examples.

**qualitative**
Left hand {{near bottom}} of play area, right hand {{just below middle}} of play area.

**ordinal**
Left hand {{on second tick mark}}, right hand {{on fourth tick mark}}.

**Quantitative**
Left hand {{on tick mark 2}}, right hand {{on tick mark 4}}. 

The design doc has been updated to reflect where this should be added in the play area.

**Comment by zepumph** [comment]
It isn't totally clear where this sentence should go. Here is what I see in the design doc:

![image](https://user-images.githubusercontent.com/6856943/91747760-37e4f000-eb6b-11ea-9dee-553108738d77.png)


Does this mean it is a sentence after the help text? Will it be the content of the "both hands" div? Part of me has always imagined this as an alert, as the "object response". Adding it to the PDOM will not provide it to the screen reader eagerly.

**Comment by zepumph** [comment]
I never reassigned, sorry.

**Comment by terracoda** [comment]
I am also confused about this issue.
@Matthew-Moore240, could you clarify?

**Comment by terracoda** [comment]
Yes, exactly, this sim currently doesn't have individual object descriptions in the Play Area. Not that it can't, but because the Both Hands interaction is not stand-alone, two step interaction, say like the Yellow Balloon in BASE, the structures may not easily provide room for a sentence.

That said, we do need a starting value for the Both Hands interaction, so I think you are right @Matthew-Moore240, we need a value to provide on focus of the interaction.

I'll paste in a screenshot of the current structure.

**Comment by terracoda** [comment]
@zepumph and @Matthew-Moore240, I suggested these same values in the brainstorming table:
Table: Initial Object Responses - Thinking through Options (DRAFTS)

We can definitely try this approach. I was concerned that describing each hand separately might be too wordy. 
I was going to suggest that we start the object value (for both hands) with distance between the hands, rather than the position of each hand. This would be for the Both Hands interaction. See the next few rows in the table.
No Tick Marks:
- Hands {{somewhat close to each other}}, Both Hands, movable

Tick Marks:
- Hands {{2 tick marks from each other}}, Both Hands, movable

Numbered Tick Marks
- Hands {{2 tick marks from each other}}, Both Hands, movable 
    - same as non-numbered tick marks

@Matthew-Moore240, what do you think of flipping the perspective to a distance focus for the both hands interaction? Apologies this not fully fleshed out, but that's what I was trying to communicate in the table.

I am still not sure what the subsequent responses would be. The above is just the initial on-focus response.

**Comment by terracoda** [comment]
At the last meeting when we were trying to brainstorm the "in between values" somebody mentioned possible describing rough number of tick marks apart. I have added the following suggestions in the design document:

- 2.5 tick marks apart or 2 and a half tick marks apart
- just over 2 tick marks apart
- 2 tick marks apart
- just under 2 tick marks apart
- 1.5 tick marks apart or 1 and a half tick marks apart
- just over 1 tick mark apart 
- 1 tick marks apart

For both hands we could use position as the supporting detail, and distance as the main detail. It could help make the object values for both hands briefer and make is somewhat different than the single hand interaction. In this way, the two interactions might be distinct from each other and also supportive of each other.

This is just a theory. What do you think?

**Comment by terracoda** [comment]
We still have the issue of where to provide the description in the interaction.
![Screen Shot 2020-09-03 at 6 46 44 PM](https://user-images.githubusercontent.com/1902116/92171048-e3b16880-ee15-11ea-8312-f0ab33e5ffab.png)

**PSEUDO-CODE**
```html
<label>Left Hand</label>
<input type="range" aria-valuetext="description-left-hand">

<label>Right Hand</label>
<input type="range" aria-valuetext="description-right-hand">

<p>Move hands up and down to find the challenge ratio in different locations.</p>

<!-- custom both hands interaction -->
<div id="both-hands" aria-role-description="movable" aria-describedby="both-hands-description">Both Hands</div>
<p or div id="both-hands-description">"description-for-both-hands"</p or div>

<p>Use W and S keys to move left hand, and Up and Down Arrow keys to move right hand. Move both hands at the same time to maintain the challenge ratio.</p>
```

Of course, this doesn't include what happens upon actual movement. It's just a scenario for on-focus.

**Comment by terracoda** [comment]
@zepumph, would that structure for both hands work?

**Comment by terracoda** [comment]
@zepumph, you'll notice that I am not sure if it matters whether a paragraph or a div is used to wrap the both hands description.

**Comment by terracoda** [comment]
@Matthew-Moore240, if that structure works, your both hands description would go in this spot:
```html
<p or div id="both-hands-description">"description-for-both-hands"</p or div>
```

Ideally, the output would be sound like:

No Tick Marks:
- {{Left hand near bottom of Play Area, right just below middle}}, Both Hands, movable

Tick Marks:
- {{Left hand on second tick mark, right on fourth tick mark}}, Both Hands, movable

Numbered Tick Marks:
- {{Left hand on tick mark 2, right on tick mark 4}}, Both Hands, movable

OR If you go with distance, it would be something like what was posted in https://github.com/phetsims/ratio-and-proportion/issues/157#issuecomment-686741206

**Comment by terracoda** [comment]
Assigning to @zepumph for comment on structure.

**Comment by zepumph** [comment]
That is very cool and clever! It think that it will work. What I don't know is if the description in the other div/p will automatically update when you are focused on the application div (which you didn't include `role=application` but I assume was meant). 

I can conduct an isolated test next time I'm on a11y.

**Comment by terracoda** [comment]
@zepumph, yes, assuming role="application" on the parent div.

And yes, there needs to be some way - maybe the same was as the PDOM updates - for the Both Hands description (i.e. it's object value) to stay in synch with the sim.

In theory, the `aria-controls` attribute would be ideal for this, but from previous research that I have read, the attribute is not very reliable.

**Comment by zepumph** [comment]
This will be picked up over in https://github.com/phetsims/ratio-and-proportion/issues/173. I will mention the use of aria-describedby over there too. Closing
